collect-settings.rst

```diff
@@ -44,15 +77,21 @@ Map settings are included on the :menuselection:`User Interface` settings page.
 Form Management Settings
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+.. image:: /img/collect-settings/form-management.*
+  :alt: Form Management settings
+  :class: device-screen-vertical
+
 :menuselection:`Form Management` settings handle default behavior when editing, finalizing, and importing forms.
 
-- :guilabel:`Auto send` When enabled, forms are sent immediately when finalized if the device can connect to the internet. You can specify whether to send over WiFi, cell data, or both.
-- :guilabel:`Delete after send` When enabled, form instances are deleted once they are sent.
+.. rubric:: Form submission
```

Comment:
I wonder if these should all be subsection headings (`""""""""""`) 
instead of rubric headings

Do you have a thought on that?
I can't remember why I used rubric headings the first time....